WebAppConfig.ps1

```diff
@@ -0,0 +1,67 @@
+param(
+	[string]$webAppTarget,
+	[string]$webAppHosts,
+	[string]$renewXNumberOfDaysBeforeExpiration = "-1", # set this to e.g. 65 after first request to avoid hitting the cert-limit if the job is triggerd manually more than 5 times
+	[string]$tenantId = "<default tenant Id>",
+	[string]$subscriptionId = "<default subscription Id>",
+	[string]$webAppLetsEncrypt = "<default letsencrypt web app>",
+	[string]$resourceGroupLetsEncrypt = "<default resource group of letsencrypt web app>",
+	[string]$clientId = "<default client Id>",
+	[string]$clientSecret = "<default client secret>",
+	[string]$resourceGroupTarget = "<default target resource group>",
+	[string]$email = "<default email>")
+
+Login-AzureRmAccount
+
+Set-AzureRmContext -SubscriptionId $subscriptionId
+
+# Load Existing Web App settings for source and target
+$webAppSource = Get-AzureRmWebAppSlot -ResourceGroupName $resourceGroupLetsEncrypt -Name $webAppLetsEncrypt -Slot "production"
+
+# Get reference to the source app settings
+$appSettingsSource = $webAppSource.SiteConfig.AppSettings
+
+# Create Hash variable for App Settings
+$appSettingsTarget = @{}
+
+# Copy over all Existing App Settings to the Hash
+ForEach ($appSettingSource in $appSettingsSource) {
+    $appSettingsTarget[$appSettingSource.Name] = $appSettingSource.Value
+}
+
+# Add new settings
+$appSettingsTarget["letsencrypt:" + $webAppTarget + "-clientId"] = $clientId
```

Comment:
Extract `letsencrypt:` to variable/constant

Reply:
Good idea!